prow/plugins/config_test.go

```diff
@@ -2147,3 +2163,57 @@ func TestMergeFrom(t *testing.T) {
 		}
 	}
 }
+
+func TestMergeExternalPluginsFrom(t *testing.T) {
```

Comment:
Could you extend `TestMergeFrom` instead of writing a new test so we don't duplicate so much testcode?